src/node_errors.h

```diff
@@ -104,6 +105,7 @@ void OnFatalError(const char* location, const char* message);
   V(ERR_TRANSFERRING_EXTERNALIZED_SHAREDARRAYBUFFER,                           \
     "Cannot serialize externalized SharedArrayBuffer")                         \
   V(ERR_TLS_PSK_SET_IDENTIY_HINT_FAILED, "Failed to set PSK identity hint")    \
+  V(ERR_WASI_NOT_STARTED, "wasi.start has not been called")                    \
```

Comment:
Would you mind making this change:

```suggestion
  V(ERR_WASI_NOT_STARTED, "wasi.start() has not been called")                  \
```